docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -425,59 +425,47 @@ Default: `3`
 
 The number of shards per YB-TServer for each YCQL table when a user table is created.
 
-Default: `-1` (the value is calculated at runtime). For servers with up to two CPU cores, the default value is considered as `4`. For three or more CPU cores, the default value is considered as `8`. Local cluster installations, created with `yb-ctl` and `yb-docker-ctl`, use a value of `2` for this flag. Clusters created with `yugabyted` use a default value of `1`.
-
-{{< note title="Important" >}}
-
-This value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
-
-{{< /note >}}
-
-{{< note title="Important" >}}
-
-When the value is set to *Default* (`-1`), then the server internally *updates* the flag with intended value during startup prior to version `2.18` and the flag remains *unchanged* starting from version `2.18`.
-
-{{< /note >}}
+Default: `-1` (the value is calculated at runtime).
```

Comment:
I'm not clear on whether this is all discussing defaults - if so, clarify as follows

The default is set depending on the type of deployment, as follows:
- For standard deployments, `-1`, which indicates that the the value is calculated at runtime as follows:
  - For servers with up to two CPU cores, the value is set to `4`.
  - For three or more CPU cores, the value is set to `8`.
- For local clusters created with `yb-ctl` or `yb-docker-ctl`, `2`.
- For clusters created with `yugabyted`, `1`.

Reply:
That's not correct, the Default is `-1` and depending on the deployment (actually num of CPUs and tablet splitting value) a server is considering on-the-fly which value should be use (before 2.18 this considered value is set as a flag's value replacing `-1` -- which is not correct enough and that's why this was changed).
These:
For local clusters created with yb-ctl or yb-docker-ctl, 2
For clusters created with yugabyted, 1
have no relation to Default.

Reply:
OK - the way it was phrased I couldn't tell. So what are they - advice?

Reply:
Current version is looking OK from my perspective.